junction/profiles/views.py

```diff
@@ -22,3 +29,26 @@ def dashboard(request):
                 conf_proposals[conf.name] = [proposal]
     return render(request, 'profiles/dashboard.html',
                   {'conf_proposals': conf_proposals})
+
+
+@login_required
+def profile(request):
+    username = request.user
+    detail = None
+
+    if request.method == "POST" and username == request.user:
+        user = get_object_or_404(User, pk=username.id)
+        detail = get_object_or_404(Profile, user=user)
+        detail_form = ProfileForm(request.POST, instance=detail)
+
+        if detail_form.is_valid():
+            detail = detail_form.save()
+            return HttpResponseRedirect(reverse('profiles:dashboard'))
+
+    elif request.method == "GET":
+        user = get_object_or_404(User, pk=username.id)
+        detail = get_object_or_404(Profile, user=user)
```

Comment:
When a user visits edit page, it is simply throwing a 404 page. Shouldnt we create a profile before if it doesnt exists for that user?